data/json/items/archery.json

```diff
@@ -318,44 +320,42 @@
          "covers" : ["TORSO"],
          "coverage" : 5,
          "material_thickness" : 1,
-         "encumbrance" : 20
+         "encumbrance" : 14
       },
-      "ranged_damage" : 3,
-      "range" : 5,
+      "reload_noise_volume": 3,
+	  "loudness": 6,
+      "ranged_damage" : 4,
+      "range" : 4,
       "dispersion" : 210,
       "durability" : 6,
       "clip_size" : 1,
-      "reload" : 60,
-      "valid_mod_locations": [["underbarrel", 1],[ "sights", 1 ],[ "accessories", 2 ]]
+      "reload" : 30
    },
    {
       "id" : "compbow",
       "type" : "GUN",
       "symbol" : "(",
       "color" : "yellow",
       "name" : "compound bow",
-      "description" : "A bow with cams for high velocity shots that can be used effectively by average archers.  Arrows fired from this weapon have a good chance of remaining intact for re-use.",
+      "description" : "A high-power bow with shaped cams and extra cables for high velocity shots that can be used effectively by fairly strong archers. Currently set to a medium weight.",
       "price" : 55000,
       "material" : ["steel", "plastic"],
-      "flags" : ["FIRE_TWOHAND", "STR_DRAW", "RELOAD_AND_SHOOT", "PRIMITIVE_RANGED_WEAPON"],
```

Comment:
Assuming STR_DRAW removed due to pulley/cam system?

Reply:
Correct. Most compounds stop at a certain draw... for some reason... drawing it past there might break something, I guess?

Reply:
According to https://www.outdoorlife.com/features/chasing-speed-fastest-compound-bow/ both
the AMO and the IBO have a maximum draw weight for compound bows of 60 and 70 pounds, respectively. I believe you're correct in that it's the capacity of the materials used in the pulley/cam system that is holding it back. Perhaps a "superalloy compound bow" should be added, that requires high skills to make, that exceeds the rest?

Reply:
I believe that's just for official testing so that all bows are comparable. I think to make a compound bow of higher weight it would just require longer or stiffer limbs, perhaps some more reinforcement in various places. You could even start looking at splitting it into two sets of limbs and cables. Problem is, 70lb is apparently enough to hunt pretty much anything, and any more would require an extremely strong person to draw.

What I considered earlier while developing my mod was a "mutant bow", basically a compound made from steel cable and car suspension leaf springs that required a minimum of 16 strength to use. Not sure how useful it would be, but it might be fun for the strength build players.

Reply:
Yes.

Reply:
A couple of new bows will come in another PR, but they'd be out of balance if added before this one.